setup.py

```diff
@@ -60,7 +60,9 @@
 def absexpanduser(x): return os.path.abspath(os.path.expanduser(x))
 
 
-VERSION = '0.7.1'
+# from pyne.pyne_version import PYNE_VERSION
```

Comment:
Looks like we can remove commented these lines in favor of those in `setup_sub.py`.